daemon/container.go

```diff
@@ -253,6 +253,7 @@ func (container *Container) Start() (err error) {
 			container.toDisk()
 			container.cleanup()
 			container.LogEvent("die")
+			container.LogEvent("exit")
```

Comment:
From looking at the code, it appears that "die" is the last event we get for a container this is being shutdown/stopped.  If we're going to have a new even that indicates something wrong happened (or at least not a normal exit) then I would prefer to see that event **before** the "die", not after.